src/effects/VST3/VST3ParametersWindow.cpp

```diff
@@ -25,6 +25,166 @@
 #include "MemoryX.h"
 #include "VST3Utils.h"
 
+
+//Interface between wx control and IEditController parameter.
+class VST3ParameterControl
```

Comment:
The commit that introduces this class fails to build on macOS.  It just needs in the header file:

#include <unordered_map>